### DISCUSSION TITLE
Tagfilter plugin stabilization

### DISCUSSION THREAD

**Discussion by Garneauma** [wet-boew/wet-boew#9551] [Provisional plugin stabilization]
Decisions on the following must be made before the plugin is stabilized:
- [ ]  The impact of the use of `[aria-controls]` from a screen-reader user perspective.
- [ ]  The use of `[aria-live]`. Whether it is applied on the element wrapping all the items or on each item individually.
- [ ]  Discuss best strategy to identify tag filtering control, by individual (like now), by group (parent element) or mixed. Or when inputs have the `.wb-tagfilter-control` class, all inputs that share its `[name]` become in scope.
- [ ]  For managing the list of tags applied on the elements, check if there is a quick way to use the DOM Token List interface to parse the content (https://dom.spec.whatwg.org/#domtokenlist) or leverage CSS selector like `[data-wb-tags~=myTag]`.

**Comment by duboisp** [comment]
Additional items:

* [ ] Ensure that all instruction is provided for each possible console.warn error throw in JS, like a default radio button must be at least a default selected value.
* [ ] Perform and publish an full ACR
* [ ] Cleanup the working example to remove the "span" that don't seem to provide any advantage, like the filtered item in the first working example.
* [ ] Optional - For the code sample, let's provide a few item being filtered and snip some the pattern repetition with "[... snip ..]". 
* [ ] Make the QA section more about wet-boew instead of GoC
